manifests/0000_30_capi-operator_00_credentials-request.yaml

```diff
@@ -0,0 +1,95 @@
+apiVersion: cloudcredential.openshift.io/v1
+kind: CredentialsRequest
+metadata:
+  name: openshift-cluster-api-aws
+  namespace: openshift-cloud-credential-operator
+  annotations:
+    exclude.release.openshift.io/internal-openshift-hosted: "true"
+    include.release.openshift.io/self-managed-high-availability: "true"
+spec:
+  secretRef:
+    name: aws-cloud-credentials
+    namespace: openshift-cluster-api
```

Comment:
does this exist?

Reply:
yes, I included this operator in the release. From now, everything in manifests directory is created by CVO, including namespace.

Reply:
I was asking, "does the secret exist". I assume this needs to be copied from openshift-machine-api

Reply:
no, the secret should be created by CCO